dune-project

```diff
@@ -28,10 +28,18 @@
 
 ; Opam packages
 
+(package
```

Comment:
why a new package? Why not part of libs/commons/ like the other Arg_helpers.ml, Logger_helpers.ml, Cmdliner_helpers.ml (that we could rename to Xxx_ext.ml)

Reply:
I'm determined to make this a standalone library that others can use as soon as possible, with the hope that some or all of it will be integrated into Alcotest eventually. For now, having it in its own folder without external dependencies other than Alcotest makes it easy for outsiders to review it.